### PR TITLE
font color enhancement

### LINKED ISSUE #629: Add Font Colour Enhancement Option in Text to PDF fragment
### What to do?
Add Font Colour Enhancement Option in Text to PDF fragment.

* Add New Enhancement Option
* Open colour chooser dialog on click
* Set the selected colour as Font colour for pdf.
* Add a checkbox `Set as default` also in the dialog.
* Set selected colour as default when `checkbox` is checked.
* Colour should be `Black` by default.

### PR DESCRIPTION
**Pull request by Ni3verma**
# Description

added font color enhancement in text to pdf

Fixes #629 

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by Swati4star** [comment]
@Ni3verma  Resolve the conflicts

**Comment by Ni3verma** [comment]
@Swati4star done

File: app/src/main/java/swati4star/createpdf/fragment/TextToPdfFragment.java [hunk 1]
```diff
@@ -373,9 +402,12 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
...
                     if (fileName != null) {
-                        if (fileName.endsWith(Constants.textExtension)) mFileExtension = Constants.textExtension;
-                        else if (fileName.endsWith(Constants.docxExtension)) mFileExtension = Constants.docxExtension;
-                        else if (fileName.endsWith(Constants.docExtension)) mFileExtension = Constants.docExtension;
+                        if (fileName.endsWith(Constants.textExtension))
+                            mFileExtension = Constants.textExtension;
+                        else if (fileName.endsWith(Constants.docxExtension))
+                            mFileExtension = Constants.docxExtension;
+                        else if (fileName.endsWith(Constants.docExtension))
+                            mFileExtension = Constants.docExtension;
```
Comment (srv-twry) [inline app/src/main/java/swati4star/createpdf/fragment/TextToPdfFragment.java]: Looks like a totally unrelated change. Why did you make this change?

Comment (Ni3verma) [reply app/src/main/java/swati4star/createpdf/fragment/TextToPdfFragment.java]: I didn't 😩
Auto reformatting by android studio. Will revert it

**Comment by Swati4star** [comment]
@vyankatesh24 merge it also
